common/djangoapps/track/backends/tests/test_logger.py

```diff
@@ -47,6 +47,22 @@ def test_logger_backend(self):
         self.assertEqual(saved_events[0], unpacked_event)
         self.assertEqual(saved_events[1], unpacked_event)
 
+    def test_logger_backend_unicode_character(self):
+        """
+        When event information contain unicode characters
+        """
+        self.handler.reset()
```

Comment:
Do you need to reset the handler at the end of the test, as well? Is this something that should go into the test setup or teardown?